Adding missing info in Order Status doc (#906)
magento/merchdocs: src/sales/order-status.md

## Predefined Order Status

|Order Status|Status Code||
|--- |--- |--- |

> |Processing|processing|When the state of new orders is set to “Processing”, the _Automatically Invoice All Items_ option becomes available in the configuration.|

Reviewer: We are avoiding the use of quotes for literal values.

```suggestion
|Processing|processing|When the state of new orders is set to `Processing`, the _Automatically Invoice All Items_ option becomes available in the configuration.|
```

Author: Done!

> |Suspected Fraud|fraud|Sometimes orders paid via PayPal, eway or any other payment gateway, mark the orders as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|

Reviewer: Same as previous PR, eway is removed from 2.4

```suggestion
|Suspected Fraud|fraud|Sometimes orders paid via PayPal or another payment gateway are marked as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```

Author: Done!

> |Pending Payment|pending_payment|This is the status used if order is created and PayPal or similar payment method is used. This means that customer has been sent to payment gateway website but no return information has been received yet. This status will change when customer pays.|

Reviewer: ```suggestion
|Pending Payment|pending_payment|This is the status used if order is created and PayPal or similar payment method is used. This means that the customer was directed to the payment gateway website, but no return information has been received yet. This status will change when customer pays.|
```

Author: Done!

> |Payment Review|payment_review|This status appear when PayPal payment review is turned on.|

Reviewer: ```suggestion
|Payment Review|payment_review|This status appears when PayPal payment review is turned on.|
```

Author: Done!

|Pending|pending|This status means no invoice and shipments have been submitted.|
|On Hold|holded|This status can only be assigned manually. You can put any order on hold.|
|Open|STATE_OPEN|

> |Complete|complete|This status means that order is crated, paid and shipped to customer.|

Reviewer: ```suggestion
|Complete|complete|This status means that the order is created, paid, and shipped to customer.|
```

Author: Done!

> |Closed|closed|This status indicates that an order have had a credit memo assigned to it and the customer has received a refund.|

Reviewer: ```suggestion
|Closed|closed|This status indicates that an order was assigned a credit memo and the customer has received a refund.|
```

Author: Done!

> |Canceled|canceled|This status is assigned manually by you or, on some payment gateways, when customer does not pay within the specified time.|

Reviewer: ```suggestion
|Canceled|canceled|This status is assigned manually in the Admin or, for some payment gateways, when the customer does not pay within the specified time.|
```

Author: Done!

> |PayPal Canceled Reversal|paypay_canceled_reversal|This status means that Paypal cancelled the reversal.|

Reviewer: ```suggestion
|PayPal Canceled Reversal|paypay_canceled_reversal|This status means that PayPal canceled the reversal.|
```

Author: Done!

> |Pending PayPal|pending_paypal|This status means that order has been received by PayPal, but payment has not yet been processed.|

Reviewer: ```suggestion
|Pending PayPal|pending_paypal|This status means that the order was received by PayPal, but payment has not yet been processed.|
```

Author: Done!

> |PayPal Reversed|paypal_reversed|Reversal means that PayPal reversed the transaction.|

Reviewer: ```suggestion
|PayPal Reversed|paypal_reversed|This status means that PayPal reversed the transaction.|
```

Author: Done!
